scanapp/tests/test_serializer.py

```diff
@@ -0,0 +1,283 @@
+            scan_directory='media/username',
+            scancode_notice='Some dummy notice',
+            scancode_version='2.0.0rc3',
+            files_count=200,
+            scan_start_time=timezone.now(),
+            scan_end_time=timezone.now()
+        )
+        scanned_file = ScannedFile.objects.create(scan=scan, path='/home/nexb/server/')
+        license = License.objects.create(
+            scanned_file=scanned_file,
+            key='A',
+            score=78,
+            short_name='mit',
+            category='some category',
+            owner='mit',
+            homepage_url='https://github.com/',
+            text_url='http://github.com/',
+            dejacode_url='https://github.com',
+            spdx_license_key='mit',
+            spdx_url='https://github.com/',
+            start_line=21,
+            end_line=567,
+            matched_rule=json.loads('{"url": ["https://github.com", "https://google.com"]}')
+        )
+        copyright = Copyright.objects.create(
+            scanned_file=scanned_file,
+            start_line=800,
+            end_line=1000
+        )
+        copyright_holder = CopyrightHolder.objects.create(
+            copyright=copyright,
+            holder='Ranvir Singh'
+        )
+        copyright_statement = CopyrightStatement.objects.create(
+            copyright=copyright,
+            statement='mit copyright statement'
+        )
+        copyright_author = CopyrightAuthor.objects.create(
+            copyright=copyright,
+            author='Ranvir Singh'
+        )
+        package = Package.objects.create(scanned_file=scanned_file, package='bootstrap')
+        scan_error = ScanError.objects.create(
+            scanned_file=scanned_file,
+            scan_error='Integration Issue'
+        )
+
+        scan_serializer_helper = AllModelSerializerHelper(scan)
+        scan_serializer = AllModelSerializer(scan_serializer_helper)
+        scan_serializer_data = """{
+            "scan": {
+                "id": 13,
+                "url": "https;//github.com",
+                "scan_directory": "media/username",
+                "scancode_notice": "Some dummy notice",
+                "scancode_version": "2.0.0rc3",
+                "files_count": 200,
+                "user": 12
+            },
+            "scanned_file": [{
+                "path": "/home/nexb/server/"
+            }],
+            "license": [{
+                "key": "A",
+                "score": 78,
+                "short_name": "mit",
+                "category": "some category",
+                "owner": "mit",
+                "homepage_url": "https://github.com/",
+                "text_url": "http://github.com/",
+                "dejacode_url": "https://github.com",
+                "spdx_license_key": "mit",
+                "spdx_url": "https://github.com/",
+                "start_line": 21, "end_line": 567,
+                "matched_rule": {"url": ["https://github.com", "https://google.com"]}
```

Comment:
I am not sure you tests data represent anything real, are they?
It might be best to use something kinda real IMHO